Formula/consul-backinator.rb

```diff
@@ -0,0 +1,31 @@
+class ConsulBackinator < Formula
+  desc "Consul backup and restoration application"
+  homepage "https://github.com/myENA/consul-backinator"
+  url "https://github.com/myENA/consul-backinator.git",
+      :tag => "v1.2",
+      :revision => "97c3d8506f332ba86b3af922cc20adb734c81dbc"
```

Comment:
Great, thanks. Typically tag+revision is only permissible if it's required, which is usually the case only if there are git submodules.